src/_pytest/python.py

```diff
@@ -521,11 +518,62 @@ def _genfunctions(self, name: str, funcobj) -> Iterator["Function"]:
                 )
 
 
+def importtestmodule(
```

Comment:
Refactor: moving this code from a method to a standalone function because we need to reuse it. No logic changes.